### ISSUE TITLE
Issues with react-scripts 1.0.x

### ISSUE THREAD

**Issue by parse** [ayroblu/ssr-create-react-app-v2#2]
After updating the react-scripts to 1.0.1, I can not start the SSR. Any ideas on how I can get this rolling? 

```
/Users/parse/prog/photonic/ssr-create-react-app-v2/src/store.js:1
(function (exports, require, module, __filename, __dirname) { import { createStore, applyMiddleware, compose } from 'redux';
                                                              ^^^^^^
SyntaxError: Unexpected token import
    at Object.exports.runInThisContext (vm.js:76:16)
```

**Comment by parse** [comment]
I managed to solve it by using:
```
require('babel-register')({
  ignore: /\/(build|node_modules)\//,
  presets: ['es2015', 'react-app']
})
```

Instead of 

`require('babel-register')({ ignore: /\/(build|node_modules)\//, presets: ['react-app'] })
`

**Comment by ayroblu** [comment]
Yeah, looks like they don't bundle it along anymore, env, is the one I'd use, and I'll merge one of the pull requests on that soon hopefully

**Comment by vitthalr** [comment]
@parse: - but this breaks the server side rendering.

The page is no longer rendered on server :(, is it working for you??
`
require('babel-register')({
  ignore: /\/(build|node_modules)\//,
  presets: ['es2015', 'react-app']
})
`
Cheers

**Comment by ayroblu** [comment]
Solved in #3
